conf/application.conf

```diff
@@ -108,29 +91,10 @@ controllers {
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
   enabled = true
```

Comment:
This can be removed too - we don't need metrics in local tests, and its set to true in app-config-common.

Reply:
👍